jina/enums.py

```diff
@@ -429,6 +429,17 @@ def is_scipy_stackable(self) -> bool:
         return self.value in [1, 2]
 
 
+class RemoteWorkspaceState(str, Enum):
```

Comment:
simply use `BetterEnum`?

Reply:
This gets stored in `workspace_store` and gets sent in response as a "string". Using `BetterEnum` would store it as an int, which might not make sense to the client.